schema/transportation/segment.yaml

```diff
@@ -147,18 +147,16 @@ properties:
           description: Physical surface of the road. May either be
             specified as a single global value for the segment, or as
             an array of surface rules.
-          oneOf:
```

Comment:
don't we require similar fix for flags?

Reply:
yup I will have another PR out for the flags fix too. We decided to break this issue up into multiple PR's